### PR TITLE
beat name in YAML should be set actual name #2880

### PR DESCRIPTION
**Pull request by YaSuenag**
Fix for #2880 .
This fix is suggested by @ruflin 
https://twitter.com/ruflin/status/793348937625309184

### CODE REVIEW

File: metricbeat/scripts/config_collector.py [hunk 1]
```diff
@@ -28,22 +24,24 @@
...
+def collect(beat_name, beat_path, full=False):
 
     base_dir = beat_path + "/module"
     path = os.path.abspath(base_dir)
 
     # yml file
 
     if full:
-        config_yml = header_full
+        config_yml = "########################## " + beat_name + " Configuration ###########################" + header_full
```
Comment (ruflin) [inline metricbeat/scripts/config_collector.py]: We should here capitalize the first letter with `.title()`.

Comment (ruflin) [inline metricbeat/scripts/config_collector.py]: I tested the branch locally and these changes are needed:
```
     if full:
-        config_yml = "########################## " + beat_name + " Configuration ###########################" + header_full
+        config_yml = "########################## " + beat_name.title() + " Configuration ###########################\n" + header_full
     else:
-        config_yml = "###################### " + beat_name + " Configuration Example #######################" + header_short
+        config_yml = "###################### " + beat_name.title() + " Configuration Example #######################\n" + header_short
```

See also the newline I added at the end. With these changes, also the build should go green.

**Comment by YaSuenag** [comment]
@ruflin Thank you for your comment!
I fixed the error. Could you review again?

**Comment by ruflin** [comment]
@YaSuenag Merged. Thanks a lot for the contribution. Now I'm curious if it will work with your beat accordingly.

**Pull request merged**